google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -265,41 +261,6 @@ public void deleteCredentials() {
   }
 
   @NotNull
-  private DeploymentOperationCallback wrapCallbackForUsageTracking(
-      final DeploymentOperationCallback deploymentCallback,
-      AppEngineDeploymentConfiguration deploymentConfiguration,
-      AppEngineEnvironment environment) {
-
-    StringBuilder labelBuilder = new StringBuilder();
-    if (environment == AppEngineEnvironment.APP_ENGINE_STANDARD) {
-      labelBuilder.append("standard");
-    } else {
-      labelBuilder.append("flex");
-
-      if (deploymentConfiguration.isAuto()) {
-        labelBuilder.append(".auto");
-      } else {
-        labelBuilder.append(".custom");
-      }
-    }
-    final String eventLabel = labelBuilder.toString();
-
-    return new DeploymentOperationCallback() {
-      @Override
-      public Deployment succeeded(@NotNull DeploymentRuntime deploymentRuntime) {
-        UsageTrackerProvider.getInstance()
```

Comment:
I was thinking about that too. I think cancellation definitely sounds like something good for this (since its user initiated). But I'm not totally sure myself if success vs failure makes sense? Hard failures will be caught and reported to us anyway.